libs/platform/ebpf_serialize.c

```diff
@@ -4,25 +4,61 @@
 // This file contains function implementations for serializing and de-serializing
 // various eBPF structures to/from ebpf_operation*_request/response structures.
 #include "ebpf_serialize.h"
+#include "ebpf_program_types.h"
+
+/**
+ * @brief Serialized program type descriptor.
+ */
+typedef struct _ebpf_serialized_program_type_descriptor
+{
+    size_t size;
+    ebpf_context_descriptor_t context_descriptor;
+    GUID platform_specific_data;
```

Comment:
I think we can the name "program_type" instead of "platform_specific_data" here?

"platform_specific_data" is used in the verifier code, as it can run both on Linux and Windows.

Reply:
Agree, good point